core/trino-main/src/main/java/io/trino/connector/informationschema/InformationSchemaPageSource.java

```diff
@@ -272,11 +273,17 @@ private void addTablesRecords(QualifiedTablePrefix prefix)
     {
         Set<SchemaTableName> tables = listTables(session, metadata, accessControl, prefix);
         Set<SchemaTableName> views = listViews(session, metadata, accessControl, prefix);
-        // TODO (https://github.com/trinodb/trino/issues/8207) define a type for materialized views
+        Set<SchemaTableName> materializedViews = listMaterializedViews(session, metadata, accessControl, prefix);
```

Comment:
put before `views =` line.
for consitency across code base, we handle mvs first